runtime/altair/src/lib.rs

```diff
@@ -353,22 +352,22 @@ impl InstanceFilter<Call> for ProxyType {
 				// Call::Democracy(..) |
 				Call::Council(..) | Call::Elections(..) | Call::Utility(..)
 			),
-			// ProxyType::Staking => matches!(c,
-			//     Call::Staking(..) |
-			//     Call::Session(..) |
-			// 	Call::Utility(..)
-			// ),
-			// ProxyType::Vesting => matches!(c,
-			//     Call::Staking(..) |
-			//     Call::Session(..) |
-			//     Call::Democracy(..) |
-			// 	Call::Council(..) |
-			// 	Call::Elections(..) |
-			// 	Call::Vesting(pallet_vesting::Call::vest(..)) |
-			// 	Call::Vesting(pallet_vesting::Call::vest_other(..))
-			// ),
+			ProxyType::_Staking => false,
+			ProxyType::NonProxy => !matches!(
+				c,
+				Call::Proxy(pallet_proxy::Call::add_proxy(..))
+					| Call::Proxy(pallet_proxy::Call::remove_proxy(..))
+					| Call::Proxy(pallet_proxy::Call::remove_proxies(..))
+					| Call::Proxy(pallet_proxy::Call::anonymous(..))
+					| Call::Proxy(pallet_proxy::Call::kill_anonymous(..))
+					| Call::Proxy(pallet_proxy::Call::announce(..))
+					| Call::Proxy(pallet_proxy::Call::remove_announcement(..))
+					| Call::Proxy(pallet_proxy::Call::reject_announcement(..))
+					| Call::Proxy(pallet_proxy::Call::proxy_announced(..))
+			),
 		}
 	}
+
 	fn is_superset(&self, o: &Self) -> bool {
```

Comment:
This needs to be updated with my fix from https://github.com/centrifuge/centrifuge-chain/pull/451/commits/f21de28bcb9cdd091752a1efa2c30383fcee3b15